daliuge-engine/dlg/apps/pyfunc.py

```diff
@@ -393,7 +393,7 @@ def run(self):
         # Inputs are un-pickled and treated as the arguments of the function
         # Their order must be preserved, so we use an OrderedDict
         if DropParser(self.input_parser) is DropParser.PICKLE:
-            all_contents = lambda x: pickle.loads(droputils.allDropContents(x))
+            all_contents = lambda x: pickle.loads(x)
```

Comment:
This diff ended up causing dask_emulation tests to timeout. I didn't think it was even used there